packages/editor/src/components/global-styles-provider/index.js

```diff
@@ -146,13 +146,30 @@ function useGlobalStylesUserConfig() {
 
 function useGlobalStylesBaseConfig() {
 	const baseConfig = useSelect( ( select ) => {
-		const { __experimentalGetCurrentThemeBaseGlobalStyles, canUser } =
-			select( coreStore );
+		const {
+			__experimentalGetCurrentThemeBaseGlobalStyles,
+			getCurrentTheme,
+			canUser,
+		} = select( coreStore );
+
+		const canReadActiveTheme = canUser( 'read', 'themes?status=active' );
```

Comment:
So to prevent a flash of global styles in the post editor, we'll need to preload these paths as well?